src/sphinxcontrib/towncrier/__init__.py

```diff
@@ -105,6 +105,21 @@ def _get_draft_version_fallback(strategy: str, sphinx_config: Dict[str, Any]):
     return ' '.join(msg_chunks)
 
 
+def _nodes_from_rst(state: statemachine.State, rst: str) -> List[nodes.Node]:
+    """Turn an RST string into a node that can be used in the document."""
+    node = nodes.Element()
+    node.document = state.document
+    nested_parse_with_titles(
+        state=state,
+        content=statemachine.ViewList(
+            statemachine.string2lines(rst),
+            source='towncrier-fragments',
```

Comment:
I'd prefer this in square brackets to indicate that it's not an input from some specific file on disk...
```suggestion
            source='[towncrier-fragments]',
```

Reply:
In reality, I’d like to know how this is used and if we can actually set it to something relevant

Reply:
AFAIU it's only used in logs unless I missed something.